.scripts/regeneration.js

```diff
@@ -10,7 +11,7 @@ const swaggerDir = 'src/node_modules/@microsoft.azure/autorest.testserver/swagge
 const goMappings = {
     'additionalpropsgroup': 'additionalProperties.json',
     'arraygroup': 'body-array.json',
-    'azurereportgroup': 'azure-report.json',
+    //'azurereportgroup': 'azure-report.json', used for reporting test coverage
```

Comment:
Why arent we including azurereportgroup and reportgroup?

Reply:
They're used as part of the `covreport` tool, so I moved the packages under its directory.  Unfortunately we can't specify custom output directories at present.

This will likely change again with the changes for generating data-plane packages as I would like to generate these packages in-place and not under a subdirectory.  It also means we'll need the ability to specify the open-api type per swagger as I know that some of them we want to treat as ARM (LROs maybe others?).